src/parser/waypoint.rs

```diff
@@ -93,69 +89,44 @@ pub fn consume<R: Read>(context: &mut Context<R>, tagname: &'static str) -> Resu
+                            Some(string::consume(context, "geoidheight", false)?.parse()?)
                     }
-                    "sat" => {
-                        waypoint.sat =
-                            Some(string::consume(context, "sat", false)?.parse().chain_err(
-                                || "error while casting number of satellites (sat) to u64",
-                            )?)
+                    "sat" => waypoint.sat = Some(string::consume(context, "sat", false)?.parse()?),
+                    "hdop" => {
+                        waypoint.hdop = Some(string::consume(context, "hdop", false)?.parse()?)
+                    }
+                    "vdop" => {
+                        waypoint.vdop = Some(string::consume(context, "vdop", false)?.parse()?)
+                    }
+                    "pdop" => {
+                        waypoint.pdop = Some(string::consume(context, "pdop", false)?.parse()?)
                     }
-                    "hdop" => waypoint.hdop = Some(
-                        string::consume(context, "hdop", false)?
-                            .parse()
-                            .chain_err(|| {
-                                "error while casting horizontal dilution of precision (hdop) to f64"
-                            })?,
-                    ),
-                    "vdop" => waypoint.vdop = Some(
-                        string::consume(context, "vdop", false)?
-                            .parse()
-                            .chain_err(|| {
-                                "error while casting vertical dilution of precision (vdop) to f64"
-                            })?,
-                    ),
-                    "pdop" => waypoint.pdop = Some(
-                        string::consume(context, "pdop", false)?
-                            .parse()
-                            .chain_err(|| {
-                                "error while casting position dilution of precision (pdop) to f64"
-                            })?,
-                    ),
                     "ageofdgpsdata" => {
-                        waypoint.dgps_age = Some(
-                            string::consume(context, "ageofdgpsdata", false)?
-                                .parse()
-                                .chain_err(|| "error while casting age of DGPS data to f64")?,
-                        )
+                        waypoint.dgps_age =
+                            Some(string::consume(context, "ageofdgpsdata", false)?.parse()?)
                     }
                     "dgpsid" => {
-                        waypoint.dgpsid = Some(
-                            string::consume(context, "dgpsid", false)?
-                                .parse()
-                                .chain_err(|| "error while casting DGPS station ID to u16")?,
-                        )
+                        waypoint.dgpsid = Some(string::consume(context, "dgpsid", false)?.parse()?)
```

Comment:
Yep, no idea why it's untouched (110 columns).

Reply:
Who knows 🤷 

I care less about the particular formatting, and just wanted to make sure it was settled in the relevant commits rather than getting formatted later in some unrelated change.